bitnami/keycloak/Chart.yaml

```
@@ -34,4 +34,4 @@ maintainers:
 name: keycloak
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/keycloak
-version: 24.5.7
\ No newline at end of file
+version: 24.5.8
```

Please bump minor version instead given you're introducing new parameters in the chart:
```suggestion
version: 24.6.0
```